minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -205,13 +175,13 @@ public void start() throws IOException {
     mWorkerConf.set(Constants.WORKER_DATA_FOLDER, mWorkerDataFolder);
     mWorkerConf.set(Constants.WORKER_MEMORY_SIZE, Long.toString(mWorkerCapacityBytes));
     mWorkerConf.set(Constants.WORKER_TO_MASTER_HEARTBEAT_INTERVAL_MS, "15");
-    mWorkerConf.set(Constants.WORKER_MIN_WORKER_THREADS, Integer.toString(1));
-    mWorkerConf.set(Constants.WORKER_MAX_WORKER_THREADS, Integer.toString(100));
-    mWorkerConf.set(Constants.WORKER_NETTY_WORKER_THREADS, Integer.toString(2));
+    mWorkerConf.set(Constants.WORKER_MIN_WORKER_THREADS, "1");
+    mWorkerConf.set(Constants.WORKER_MAX_WORKER_THREADS, "100");
+    mWorkerConf.set(Constants.WORKER_NETTY_WORKER_THREADS, "2");
 
     // Perform immediate shutdown of data server. Graceful shutdown is unnecessary and slow
-    mWorkerConf.set(Constants.WORKER_NETTY_SHUTDOWN_QUIET_PERIOD, Integer.toString(0));
-    mWorkerConf.set(Constants.WORKER_NETTY_SHUTDOWN_TIMEOUT, Integer.toString(0));
+    mWorkerConf.set(Constants.WORKER_NETTY_SHUTDOWN_QUIET_PERIOD, "0");
+    mWorkerConf.set(Constants.WORKER_NETTY_SHUTDOWN_TIMEOUT, "0");
```

Comment:
No particular reason, just because mixed styles in this piece, so I changed them all to string. Will change them all back using explicit conversion.